lib/svg-to-sprite.js

```diff
@@ -0,0 +1,25 @@
+const sprites = require('svgstore')();
```

Comment:
This import should be last.

Reply:
Why ? 🤔

Reply:
It's the alphabet's fault.

Reply:
Oh, right. I thought they were a more technical reason 😅